src/clients/helloworld.rb

```diff
@@ -7,4 +7,9 @@
 
 require "y2helloworld/main_module"
 
-Y2HelloWorld::MainDialog.new.run
+if ARGV.any?("help")
```

Comment:
this is not the standard way and also it is not translated. To see inspiration how it is done in other modules see https://github.com/yast/yast-packager/blob/d4a21e0ac882b23b68a935367b89e8a1ca6fb273/src/clients/key_manager.rb#L22